### ISSUE TITLE
auto-merge label not set on docs cherrypicks

### ISSUE THREAD

**Issue by craigbox** [sebastienvas/istio-testing#4140]
https://github.com/istio/istio.io/#publishing-content-immediately says that when the `cherrypick/release-1.14` label is set on an istio/istio.io PR, it should be merged automatically.  

The label itself says "Set this label on a PR to auto-merge it to the release-1.14 branch'.

The auto-merge label isn't set on the PRs created by the cherrypicker (https://github.com/istio/test-infra/blob/master/prow/cluster/cherrypicker_deployment.yaml#L20-L29).

@ericvn assumes this is because we only want this behavior on the docs repo, and if the `auto-merge` label was set here, it would apply project-wide? 

I can't talk for the other repos, but as the docs repo needs to cherrypick almost every change, it would be a great quality of life improvement to change this here.

**Comment by ericvn** [comment]
Correct, adding the label there would apply to all repos, which we don't want. I believe we just need to create another cherry-picker for the istio.io repo only which adds the additional label (and remove istio.io from the current one)..

**Comment by dhawton** [comment]
Is this something that is still needed?

**Comment by craigbox** [comment]
It would be nice, yes!

**Comment by howardjohn** [comment]
I am not sure it's safe to allow arbitrary people to decide arbitrary
content should go to arbitrary branches without review

**Comment by howardjohn** [comment]
Actually thinking about it more - this is extremely unsafe, we cannot do
this without some additional restrictions.

**Comment by ericvn** [comment]
I agree for nearly all repos, we do not want auto merge on cherry-picks for the reasons John stated. My original statement was that the behavior today is org wide, and thus we shouldn't make the change. Craig is asking about istio.io only, and for that I'm less particular about the auto-merge on the cherry-pick since it's typically doc related and for the most part, the approval is pretty automatic anyway. A reason not to do it for istio.io, would be the typical auto-merge would be into the current production branch (the istio.io website) and that's very visible very quickly after the merge.

**Comment by craigbox** [comment]
The reason _to_ do it is because the reality is we write most of the docs for 1.2x when 1.2x is out, rather than doing it all beforehand and freezing it like we would in a perfect world.

Another possible reason not to do it would be that anyone could set `cherry-pick: release-1.0` on any PR and it would be cherry-picked to a really old, nonsensical branch.  

I would like to get to a world where we only have the docs for supported Istio releases online...
